docs/configuration.adoc

```diff
@@ -57,15 +57,15 @@ instead of *new* ones.
 
 In this strategy not committed changes (those that are _modified_ or _changed_) are considered as important tests as well.
 
-TIP: Usually `new` and `changed` strategies are used together `-Dconst:core/src/main/java/org/arquillian/smart/testing/Configuration.java[name="SMART_TESTING"]=new, changed`.
+TIP: Usually `new` and `changed` strategies are used together `-Dconst:core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java[name="SMART_TESTING"]=new, changed`.
 
 ==== Affected
 
 `Affected` strategy uses a different approach to choose what are the *important* tests to run first (_ordering_) or filtered (_selecting_).
 This strategy also relies on _SCM_ information but in this case it retrieves any *new* or *modified* business class
 between commits range and local changes as well.
 
-TIP: `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="COMMIT"]`, `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="PREVIOUS_COMMIT"]` and `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]` Java system properties are valid as well.
+TIP: `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="SCM_RANGE_HEAD"]`, `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="SCM_RANGE_TAIL"]` and `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="DEFAULT_LAST_COMMITS"]` Java system properties are valid as well.
```

Comment:
I guess that the `DEFAULT_LAST_COMMITS` is wrong - there should be the system property used...